language/gpt-j/README.md

```diff
@@ -75,6 +73,16 @@ Evaluates the ROGUE scores from the accuracy logs. Only applicable when specifiy
 ```
 python evaluation.py --mlperf-accuracy-file ./build/logs/mlperf_log_accuracy.json --dataset-file ./data/cnn_eval.json
 ```
+
+### Reference Model - ROUGE scores
+The following are the rouge scores obtained when evaluating the fp32 model on the entire test set using greedy search (11490 samples)
```

Comment:
This is really confusing for submitters and contradictory to what you wrote in [https://github.com/mlcommons/inference/issues/1384#issuecomment-1571088874](https://github.com/mlcommons/inference/issues/1384#issuecomment-1571088874).

Please report results on the validation set only. I assume that in README we publish accuracy reference numbers so they should be obtained on the dataset json generated by download_cnndm.py.

I think the whole confusion comes from the fact that internal version of this script uses "test" instead of "validation".

I don't think we have to specify "using greedy search". Generation algorithm cannot be changed in the closed division.

Reply:
I've updated the ROUGE scores obtained using the validation set. It is important to specify greedy search because we decided to go with beam search only recently. Although we don't have the complete numbers using beam search, for any subset of the dataset, we didn't notice a significant change in the scores.